### PR TITLE
display proper error message when importing not well-formed workflow

### PR DESCRIPTION
**Pull request by Aminelouati**

### CODE REVIEW

File: app/scripts/proactive/rest/studio-client.js [hunk 1]
```diff
@@ -265,8 +265,12 @@ define(
         validateWithPopup: function (jobXml, jobModel, automaticValidation) {
             if (!localStorage['pa.session']) return;
             
-            if (automaticValidation && (jobModel.getTasksCount() == 0)) return;
+            if ( automaticValidation) {
```
Comment (marcocast) [inline app/scripts/proactive/rest/studio-client.js]: why this change?

Comment (Aminelouati) [reply app/scripts/proactive/rest/studio-client.js]: In javascript the evaluation of a boolean expression is atomic. When we import a not well-formed workflow, the job model is null in this case and therefore, jobModel.getTasksCount()  throws an exception. So this is a workaround to avoid this from happening.
